LunrCoreLmdb/Globalization/fr/LmdbIndex.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Lunr;
+using Lunr.Globalization.fr;
+
+namespace LunrCoreLmdb.Globalization.fr
+{
+    public static class LmdbIndex
+    {
+        /// <summary>
+        /// A convenience function for configuring and constructing
+        /// a new lunr DelegatedIndex.
+        /// 
+        /// An `LmdbBuilder` instance is created and the pipeline setup
+        /// with a trimmer, stop word filter and stemmer.
+        /// 
+        /// This builder object is yielded to the configuration function
+        /// that is passed as a parameter, allowing the list of fields
+        /// and other builder parameters to be customized.
+        /// 
+        /// All documents _must_ be added within the passed config function.
+        /// </summary>
+        /// <example>
+        /// var idx = Index.Build(async builder =>
+        /// {
+        ///      builder
+        ///         .AddField("title")
+        ///         .AddField("body");
+        /// 
+        ///      builder.ReferenceField = "id";
+        /// 
+        ///      foreach(Document doc in documents)
+        ///      {
+        ///          builder.add(doc);
+        ///      }
+        /// });
+        /// </example>
+        /// <param name="path">The directory path to the LMDB database used to store this index.</param>
+        /// <param name="trimmer">An optional trimmer. Default is a regex-based word trimmer.</param>
+        /// <param name="stopWordFilter">An optional stopword filter. Default is English.</param>
+        /// <param name="stemmer">An optional stemmer. Default is English.</param>
+        /// <param name="config">A Configuration function.</param>
+        /// <param name="tokenizer">An optional tokenizer. Default is a ToString() based splitter. </param>
+        /// <param name="registry">An optional pipeline function registry. Default filters through the specific trimmer, stopword filter, and stemmer.</param>
+        /// <param name="indexingPipeline">An optional indexing pipeline. Default filters through the specific trimmer, stopword filter, and stemmer. </param>
+        /// <param name="searchPipeline">An optional search pipeline. Default filters through the stemmer.</param>
+        /// <param name="cancellationToken">An optional cancellation token. Default is equivalent to CancellationToken.None.</param>
+        /// <param name="fields">The fields for this builder.</param>
+        /// <returns>The delegated index.</returns>
```

Comment:
The list of params in the doc is not the same as the actual list of params.

Reply:
Fixed.